docs/source/conf.py

```diff
@@ -441,13 +452,67 @@ def check_timeout_and_api_kwargs_presence(obj: object) -> int:
     )
 
 
+def create_return_admonitions() -> dict[str, str]:
+    """Creates 'Returned in' admonitions for classes that are returned in Bot's methods."""
+
+    methods_for_class = defaultdict(list)
+
+    # checking for islower() to avoid camelCase methods
+    for method in [m for m in dir(telegram.Bot) if not m.startswith("_") and m.islower()]:
+        try:
+            sig = inspect.Signature.from_callable(getattr(telegram.Bot, method))
+        except TypeError:  # not a method
+            continue
+
+        class_name = str(sig.return_annotation)
+        if "telegram" not in class_name:
+            continue
+
+        class_name = (
+            class_name.replace("typing.Union[", "")
+            .replace("typing.Tuple[", "")
+            .replace("]", "")
+            .split(", ")[0]  # Union and Tuple give relevant class before the comma
```

Comment:
Union *may* give several relevant classes though, right? e.g. `Union[ChatMemberAdmin, ChatMemberUser]`. I guess we currently only have `Union[something, None]`, but it may still be good if the code can already handle that. just in case …

Reply:
... or `Union[Something, bool]`. But I see your point

Reply:
Done